smart-contracts/contracts/cl-vault/src/reply.rs

```diff
@@ -30,24 +33,17 @@ pub enum Replies {
     Unknown,
 }
 
-pub fn handle_reply(deps: DepsMut, _env: Env, msg: Reply) -> Result<Response, ContractError> {
+pub fn handle_reply(deps: DepsMut, env: Env, msg: Reply) -> Result<Response, ContractError> {
     match msg.id.into() {
-        Replies::DepositCreatePool => todo!(),
+        Replies::DepositCreatePosition => handle_deposit_create_position(deps, env, msg.result),
```

Comment:
it would be reeally sick to rename this to handle_deposit_create_position_reply

Reply:
Yeah totes